e2e/cypress/integration/scroll/channel_scroll_spec.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @scroll
+
+describe('Scroll', () => {
+    let testTeam;
+    let testChannel;
+    let otherUser;
+
+    beforeEach(() => {
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+
+            cy.apiCreateUser().then(({user: user2}) => {
+                otherUser = user2;
+
+                cy.apiAddUserToTeam(testTeam.id, otherUser.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, otherUser.id);
+                });
+            });
+
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-T2378 Channel with only a few posts opens at the bottom', () => {
+        // # Post a starting message with user 1
+        cy.postMessage('This is the first post');
+
+        // # Post as few as messages so that scroll bar appears as channel name scroll hidden
+        Cypress._.times(8, (postIndex) => {
+            cy.postMessage(`p-${postIndex + 1}`);
+        });
```

Comment:
Currently the test fails because the text "Beginning on Channel..." is still visible because of less number of posts. Can you please increase the count from 8 to 20.